src/common/debug.c

```diff
@@ -144,7 +144,7 @@ daos_debug_set_id_cb(d_log_id_cb_t cb)
 
 /** Initialize debug system */
 int
-daos_debug_init(char *logfile)
+daos_internal_debug_init(char *logfile, d_dbug_t logmask)
```

Comment:
Minor, if ew are exposing this API in the header, we should probably do something like daos_debug_init_ex or some such.  It's just a more flexible version of the existing API.

Regardless, you need to change the daos_server.yml file to fix the default for log_mask

Reply:
Sorry Jeff I am not sure I get it, do you want me to change the new API name from daos_internal_debug_init() to daos_debug_init_ex() ??

And do you mean to change "#  log_mask: WARN" line to "#  log_mask: INFO" in utils/config/daos_server.yml template ??
BTW, it would have been a much simpler patch by itself :-)

Reply:
I'm just suggesting that having an API (even an internal one) that says "internal" is weird.  So yeah, it's not something I'd block this on but I'd suggest that name change.

Yes, changing it in the template (or anywhere else the default is documented as it's incorrect with this patch.